src/language-js/printer-estree.js

```diff
@@ -3927,25 +3927,24 @@ function printJestEachTemplateLiteral(node, expressions, options) {
       }
     }
 
-    const maxColumnCount = tableBody.reduce(
-      (maxColumnCount, row) => Math.max(maxColumnCount, row.cells.length),
-      headerNames.length
+    const maxColumnCount = Math.max(
+      headerNames.length,
+      ...tableBody.map(row => row.cells.length)
     );
 
-    const maxColumnWidths = Array.from(new Array(maxColumnCount), () => 0);
-    const table = [{ cells: headerNames }].concat(
-      tableBody.filter(row => row.cells.length !== 0)
-    );
-    table
-      .filter(row => !row.hasLineBreak)
-      .forEach(row => {
-        row.cells.forEach((cell, index) => {
-          maxColumnWidths[index] = Math.max(
-            maxColumnWidths[index],
-            getStringWidth(cell)
-          );
-        });
+    const maxColumnWidths = Array.from({ length: maxColumnCount }).fill(0);
```

Comment:
```suggestion
    const maxColumnWidths = new Array(maxColumnCount).fill(0);
```

Reply:
No , thanks, I  think this is exactly why es has `Array.from` `Array.of`.

Reply:
https://2ality.com/2018/12/creating-arrays.html#cheat-sheet%3A-creating-arrays
https://2ality.com/2018/12/creating-arrays.html#recommended-patterns

Reply:
Personally I always use `Array.from({length})` to generate array of this length, I think it's best practice.

Another PR for eslint also use it https://github.com/eslint/eslint/pull/12608/files

Reply:
they are using it in different way, they are using callback to prefill values

```suggestion
    const maxColumnWidths = Array.from({ length: maxColumnCount }, () => 0);
```

Reply:
they are using it in different way, they are using callback to prefill values

```suggestion
    const maxColumnWidths = Array.from({ length: maxColumnCount }, () => 0);
```

Reply:
they are using it in different way, they are using callback to prefill values

```js
    const maxColumnWidths = Array.from({ length: maxColumnCount }, () => 0);
```

Reply:
I wrote them differently on purpose. There is no difference in this case

Reply:
There is no they, still me in that PR.